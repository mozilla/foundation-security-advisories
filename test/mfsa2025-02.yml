# More testcases
announced: January 1, 2025
impact: low
fixed_in:
  - Firefox 128
title: Test Security Vulnerabilities fixed in Firefox 128
advisories:
  MFSA-TMP-2025-00001:
    title: Test advisory
    impact: low
    reporter: Reporter
    description: |
      Description without a dot
    bugs:
      - url: 1000000
  MFSA-TMP-2025-00002:
    title: Test advisory
    impact: low
    reporter: Reporter
    description: |
      Description. *With a notice in asterisks.*
    bugs:
      - url: 1000000
  MFSA-TMP-2025-00003:
    title: Test advisory
    impact: low
    reporter: Reporter
    description: |
      Description <code>With a code block</code>.
    bugs:
      - url: 1000000
  MFSA-TMP-2025-00004:
    title: Test advisory with dot in title.
    impact: low
    reporter: Reporter
    bugs:
      - url: 1000000
