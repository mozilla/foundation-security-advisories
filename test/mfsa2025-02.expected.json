{
  "MFSA-TMP-2025-00001": {
    "containers": {
      "cna": {
        "affected": [
          {
            "product": "Firefox",
            "vendor": "Mozilla",
            "versions": [
              {
                "lessThan": "128",
                "status": "affected",
                "version": "unspecified",
                "versionType": "custom"
              }
            ]
          }
        ],
        "credits": [
          {
            "lang": "en",
            "value": "Reporter"
          }
        ],
        "descriptions": [
          {
            "lang": "en",
            "supportingMedia": [
              {
                "base64": false,
                "type": "text/html",
                "value": "Description without a dot. This vulnerability affects Firefox < 128."
              }
            ],
            "value": "Description without a dot. This vulnerability affects Firefox < 128."
          }
        ],
        "references": [
          {
            "url": "https://www.mozilla.org/security/advisories/mfsa2025-02/"
          },
          {
            "url": "https://bugzilla.mozilla.org/show_bug.cgi?id=1000000"
          }
        ],
        "title": "Test advisory"
      }
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1"
  },
  "MFSA-TMP-2025-00002": {
    "containers": {
      "cna": {
        "affected": [
          {
            "product": "Firefox",
            "vendor": "Mozilla",
            "versions": [
              {
                "lessThan": "128",
                "status": "affected",
                "version": "unspecified",
                "versionType": "custom"
              }
            ]
          }
        ],
        "credits": [
          {
            "lang": "en",
            "value": "Reporter"
          }
        ],
        "descriptions": [
          {
            "lang": "en",
            "supportingMedia": [
              {
                "base64": false,
                "type": "text/html",
                "value": "Description. *With a notice in asterisks.*. This vulnerability affects Firefox < 128."
              }
            ],
            "value": "Description. *With a notice in asterisks.*. This vulnerability affects Firefox < 128."
          }
        ],
        "references": [
          {
            "url": "https://www.mozilla.org/security/advisories/mfsa2025-02/"
          },
          {
            "url": "https://bugzilla.mozilla.org/show_bug.cgi?id=1000000"
          }
        ],
        "title": "Test advisory"
      }
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1"
  },
  "MFSA-TMP-2025-00003": {
    "containers": {
      "cna": {
        "affected": [
          {
            "product": "Firefox",
            "vendor": "Mozilla",
            "versions": [
              {
                "lessThan": "128",
                "status": "affected",
                "version": "unspecified",
                "versionType": "custom"
              }
            ]
          }
        ],
        "credits": [
          {
            "lang": "en",
            "value": "Reporter"
          }
        ],
        "descriptions": [
          {
            "lang": "en",
            "supportingMedia": [
              {
                "base64": false,
                "type": "text/html",
                "value": "Description <code>With a code block</code>. This vulnerability affects Firefox < 128."
              }
            ],
            "value": "Description `With a code block`. This vulnerability affects Firefox < 128."
          }
        ],
        "references": [
          {
            "url": "https://www.mozilla.org/security/advisories/mfsa2025-02/"
          },
          {
            "url": "https://bugzilla.mozilla.org/show_bug.cgi?id=1000000"
          }
        ],
        "title": "Test advisory"
      }
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1"
  },
  "MFSA-TMP-2025-00004": {
    "containers": {
      "cna": {
        "affected": [
          {
            "product": "Firefox",
            "vendor": "Mozilla",
            "versions": [
              {
                "lessThan": "128",
                "status": "affected",
                "version": "unspecified",
                "versionType": "custom"
              }
            ]
          }
        ],
        "credits": [
          {
            "lang": "en",
            "value": "Reporter"
          }
        ],
        "descriptions": [
          {
            "lang": "en",
            "supportingMedia": [
              {
                "base64": false,
                "type": "text/html",
                "value": "Test advisory with dot in title. This vulnerability affects Firefox < 128."
              }
            ],
            "value": "Test advisory with dot in title. This vulnerability affects Firefox < 128."
          }
        ],
        "references": [
          {
            "url": "https://www.mozilla.org/security/advisories/mfsa2025-02/"
          },
          {
            "url": "https://bugzilla.mozilla.org/show_bug.cgi?id=1000000"
          }
        ],
        "title": "Test advisory with dot in title."
      }
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1"
  }
}